src/components/Sections/Lessons/Filters.js

```diff
@@ -46,8 +48,9 @@ const Filters = ({ namespace, baseParams, t }) => {
     }
   }, [isReady, baseParams, wip, err, namespace, dispatch]);
 
+  const selLen = selected.reduce((acc, x) => acc + x.values.length, 0);
   useEffect(() => {
-    if (isHydrated && isReady) {
+    if (isHydrated && isReady && prevSelRef.current !== selLen) {
```

Comment:
Copy paste of the previous code.... why keep duplicates?

Reply:
not clear duplicate with what?